apps/antalmanac/src/components/RightPane/SectionTable/SectionTable.tsx

```diff
@@ -168,7 +169,13 @@ function SectionTable(props: SectionTableProps) {
                     analyticsAction={analyticsEnum.classSearch.actions.CLICK_PAST_ENROLLMENT}
                     text="Past Enrollment"
                     icon={<ShowChartIcon />}
-                    redirectLink={`https://zot-tracker.herokuapp.com/?dept=${encodedDept}&number=${courseDetails.courseNumber}&courseType=all`}
+                    popupContent={
+                        <EnrollmentHistoryPopup
+                            department={courseDetails.deptCode}
+                            courseNumber={courseDetails.courseNumber}
+                            isMobileScreen={isMobileScreen}
```

Comment:
Since `isMobileScreen` is a hook usable by any component, you can just use it directly in the popup component. It's easier to manage functions with less inputs.
```suggestion
```